requestservice.js

```diff
@@ -0,0 +1,77 @@
+// overkill for now?
+class CreateRequest {
+	constructor(braid) {
+		this.braid = braid;
+	}
+}
+
+// TODO: class
+const braid = {
+  name: 'Devotional',
```

Comment:
Can we had a creatorUsername field.
And a created or createdAt field?

Reply:
Also can the user object  be added and modeled after the live service whose api works? 

You can add a list of snippet ids or objects as a field of your dummy service